case_mapping/uco/tool.py

```diff
@@ -6,9 +6,9 @@
 class Tool(ObjectEntity):
     def __init__(
         self,
-        tool_name=None,
-        tool_version=None,
-        tool_type=None,
+        tool_name: Optional[str] = None,
```

Comment:
I suggest this be delegated to `uco-core:name`'s keyword parameter as inherited by `ObjectEntity`.  See the `super()` call-updates I've been doing elsewhere, like on `InvestigativeAction` in this PR.

Reply:
@ajnelson-nist I didn't make any changes because this not clear to me, the superclass ObjectEntity includes the uco-core:description not the uco-core:name (are one an alternative to the other?)

Reply:
I added the `name` parameter [here](https://github.com/casework/CASE-Mapping-Python/pull/51/commits/b3b7a1d6d748c7e40db7b912fd94a7c7ef2aaf08#diff-72186be81e4d06c90035a7da4a2419ba50c2fe5dae98726c1c1dda4fba226efcR194).

The two properties are distinct in purpose.

Reply:
I still think the `*args: Any` / `super()` superclass call style should be applied here, with `tool_name` replaced with a delegation to `ObjectEntity`'s `__init__`.